hack/create.sh

```diff
@@ -75,7 +75,7 @@ properties:
 EOF
 
 go generate ./...
-go run cmd/createorupdate/createorupdate.go
+go run cmd/createorupdate/createorupdate.go -loglevel=debug
```

Comment:
No reason to touch these if you already default to debug?

Reply:
I prefer to be explicit than implicit - if we change the default level later, we will need those. What do you think?